pkg/model/resource/resource.go

```diff
@@ -54,13 +55,14 @@ type Resource struct {
 
 	// CRDVersion holds the CustomResourceDefinition API version used for the Resource.
 	CRDVersion string `json:"crdVersion,omitempty"`
-	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Resource.
+
+	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Data.
 	WebhookVersion string `json:"webhookVersion,omitempty"`
 }
 
-// GVK returns the group-version-kind information to check against tracked resources in the configuration file
-func (r *Resource) GVK() config.GVK {
-	return config.GVK{
+// Data returns the group-version-kind information to check against tracked resources in the configuration file
+func (r *Resource) Data() config.Data {
```

Comment:
This method should stay as `Data` even after changing the type to `ResourceData` as the `Resource` part is inplicit (`Resource` type).